babel-code-frame/babel-code-frame-tests.ts

```diff
@@ -0,0 +1,38 @@
+/// <reference path="./index.d.ts" />
+/// <reference path="../mocha/index.d.ts" />
```

Comment:
do not add `/// <reference path="..." />` the `tsconfig.json` will get you the `index.d.ts` ans the import for `chai` below will get you `chai/index.d.ts`